Jenkinsfile

```diff
@@ -7,11 +12,17 @@ def buildApps(appList) {
   }
 }
 
-def runningOn(machine) {
-  println "Stage running on:"
-  println machine
+def buildDocs(){
```

Comment:
I would like this function to contain the withVenv rather than it being outside.
The reason is that one day, someone might choose to reuse this function and the `pip install` would try and use the system python.